instrumentation/README.md

```diff
@@ -8,7 +8,7 @@ Instrumentation libraries provide pre-built OpenTelemetry instrumentation for po
 
 Released instrumentations can be found at the [OpenTelemetry registry](https://opentelemetry.io/registry/?language=ruby&component=instrumentation#).  You can also look in this project's Github repository: Individual instrumentation libraries can be found in subdirectories under `/instrumentation`.
 
-In-progress instrumentations can be found [here](https://github.com/open-telemetry/opentelemetry-ruby/issues?q=is%3Aopen+label%3Ainstrumentation+-label%3A%22help+wanted%22+).
+In-progress instrumentations can be found [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues?q=is%3Aopen+label%3Ainstrumentation+-label%3A%22help+wanted%22+).
```

Comment:
Not a blocker at all, but for dicsussion: we'll need to migrate tickets over from otel-ruby eh? Or do we want to leave them there?

Reply:
We agreed that we would relocate only open issues to `otel-ruby-contrib`

Reply:
[Notes from SIG](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit#bookmark=id.50fxhh42vs2w)